playbooks/edx_sandbox.yml

```diff
@@ -37,6 +38,10 @@
       when: "'localhost' in EDXAPP_MONGO_HOSTS"
     - { role: 'edxapp', celery_worker: True }
     - edxapp
+    - role: ecommerce
+      when: COMMON_ENABLE_ECOMMERCE
+    - role: ecomworker
+      when: COMMON_ENABLE_ECOMMERCE
```

Comment:
Makes sense, I suppose you might want to have the worker running on a separate machine.